site/docs/v1/tech/core-concepts/entity-management.adoc

```diff
@@ -155,3 +156,8 @@ When this value is omitted a unique Client secret will be generated automaticall
 
 [field]#Entity Type# [required]#Required#::
 When creating this Entity, you can assign it to a previously created Entity Type
+
+== Limitations
+
+It is currently not possible to have a users granted entity permissions returned via an OAuth Grant process.  A workaround would be to use the API to gather these details. +
```

Comment:
Typo. `users` -> `user`

Perhaps this could be re-worded, I'm not sure how to best describe it. Perhaps @voidmain has a better way to say it. 

Maybe:

> It is not currently possible to utilize an OAuth2 grant to retrieve user permissions to an entity. 

Re: a workaround, if we are going to list a work around here, we should be more specific, or you can defer to the GH issue to describe a work around.